blocks/start/start.js

```diff
@@ -125,9 +140,82 @@ class DaStart extends LitElement {
 
   async submitForm(e) {
     e.preventDefault();
-    const opts = { method: 'PUT' };
-    const resp = await daFetch(e.target.action, opts);
-    if (!resp.ok) return;
+
+    // Check if user is signed in
+    if (!window.adobeIMS?.isSignedInUser()) {
+      this._errorText = 'You need to sign in first.';
+      return;
+    }
+
+    this._loading = true;
+
+    // Check if this is a new org and add org-level permissions
+    const orgUrl = e.target.action.substring(0, e.target.action.lastIndexOf('/'));
+    const orgCheckResp = await daFetch(orgUrl);
+    if (orgCheckResp.status === 404) {
+      // Check if user has an email address
+      const { email } = await window.adobeIMS.getProfile();
+      if (!email) {
+        this._errorText = 'Make sure your profile contains an email address.';
+        return;
+      }
+
+      const configJson = {
+        data: {
+          total: 2,
+          limit: 2,
+          offset: 0,
+          data: [{ key: '', value: '' }],
+          ':colWidths': [169, 169],
+        },
+        permissions: {
+          total: 2,
+          limit: 2,
+          offset: 0,
+          data: [
+            {
+              path: 'CONFIG',
+              groups: email,
+              actions: 'write',
+              comments: 'The ability to set configurations for an org.',
+            },
+            {
+              path: '/ + **',
+              groups: email,
+              actions: 'write',
+              comments: 'The ability to create content.',
+            },
+          ],
+          ':colWidths': [169, 169, 169, 300],
+        },
+        ':names': ['data', 'permissions'],
+        ':version': 3,
+        ':type': 'multi-sheet',
+      };
+
+      const body = new FormData([['config', JSON.stringify(configJson)]]);
+      const orgResp = await daFetch(orgUrl, { method: 'PUT', body });
+      if (!orgResp.ok) {
+        if (orgResp.status === 401 || orgResp.status === 403) {
```

Comment:
Can this really happen ?

Reply:
I thought it prudent to check for... unless we fully trust the `window.adobeIMS.isSignedInUser()` above?